src/server/actions/create-shortened-url.ts

```
@@ -40,7 +40,9 @@ export default async function Action_CreateShortenedURL(
             ? await generateURLPath()
             : submission.value.alias
 
-    const title = await getPageTitleFromURL(submission.value.url)
+    const title = !!submission.value.title
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)